src/style.css

```diff
@@ -0,0 +1 @@
+/* styles */
```

Comment:
- [ ] KIndly remove commented out code, you can add some minimal styling if you wish